weather_mv/loader_pipeline/bq.py

```diff
@@ -154,6 +163,24 @@ def __post_init__(self):
         with open_dataset(self.first_uri, self.xarray_open_dataset_kwargs,
                           self.disable_grib_schema_normalization, self.tif_metadata_for_datetime,
                           is_zarr=self.zarr) as open_ds:
+
+            if self.create_polygon:
```

Comment:
I think the user-defined boolean should override the default behavior.

Reply:
Hello @alxmrs, I'd like to discuss the two approaches regarding this flag. Here are the options we have:
1. The current implementation sets the default value of [`create_polygon`](https://github.com/google/weather-tools/pull/337/files#diff-938d4c3999aa9d0bd9f832240de92dc8ea660bf71f8b9b6d751b3a16773ce4b1R98) as False. When the user passes the flag, the value becomes True, and a Polygon is created if the input file contains more than a single grid point.
2. An alternative approach is to have the default value of `create_polygon` as True. In this case, a Polygon is created based on the input file by default, and if the user passes the flag, we will not create a polygon.

Currently, I have implemented the first option in the code. I would appreciate your input on whether this approach is suitable, or if we should consider moving towards the second option.

Reply:
As discussed over VC, let's do option (2).

Reply:
Code updated as per your suggestion. Thanks for the input!